app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -19,6 +19,7 @@ def as_json
           reject_by_default_at: application_choice.reject_by_default_at&.iso8601,
           candidate: {
             id: "C#{application_form.candidate.id}",
+            application_reference: application_form.support_reference,
```

Comment:
I think this is the wrong place for the attribute as it's possible for the same candidate to have multiple applications, in the event that a) their first application fails and they go into "Apply again" or b) they Apply in the following cycle. One level up, in `attributes`, might be better?